activation/handler_v2.go

```diff
@@ -571,7 +712,6 @@ func (h *HandlerV2) syntacticallyValidateDeps(
 }
 
 func (h *HandlerV2) checkMalicious(
-	ctx context.Context,
 	tx *sql.Tx,
 	watx *wire.ActivationTxV2,
```

Comment:
Currently, it works a little differently. Here we only check if the owner of ATX is already malicious. Later we marry all IDs (regardless of malfeasance) and check again if the owner is malicious (it will be if any of the married IDs was malicious). It works because the `identities.IsMalicious()` scans the whole equivocation set (1 malicious ID in the set makes everybody malicious).

It's done this way because it's faster. It requires 2 DB calls instead of 256.